packages/ember/tests/routing/basic_test.js

```diff
@@ -3827,19 +3700,6 @@ test('Can disconnect from nested render helpers', function() {
   equal(jQuery('#qunit-fixture .bar').text(), '');
 });
 
-test('Can render with layout', function() {
```

Comment:
This is testing `layoutName` on a view. This is now tested by [these tests](https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L94-L128)